core/tauri/src/lib.rs

```diff
@@ -651,6 +674,7 @@ pub trait Manager<R: Runtime>: sealed::ManagerBase<R> {
   ///   }
   /// }
   /// ```
+  #[tracing::instrument("app.emit.rust", skip(self))]
```

Comment:
```suggestion
  #[tracing::instrument("app::emit_rust", skip(self))]
```

I don't like that the span name differs from the method name so much